src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Package/Update/PackageUpdateCommandRunner.cs

```diff
@@ -107,23 +133,30 @@ internal static async Task<int> Run(PackageUpdateArgs args, ILoggerWithColor log
         // 4. Update MSBuild files
         var projectName = Path.GetFileNameWithoutExtension(dgSpec.Projects[0].FilePath);
         logger.LogInformation($"  {projectName}:");
-        logger.LogInformation($"    " + Format.PackageUpdate_UpdatedMessage(packageToUpdate.Id, packageToUpdate.CurrentVersion.ToShortString(), packageToUpdate.NewVersion.ToShortString()));
-        logger.LogInformation("");
 
         var updatedPackageSpec = updatedDgSpec.Projects[0];
-        packageUpdateIO.UpdatePackageReference(updatedPackageSpec, restorePreviewResult, packageTfms!, packageToUpdate, logger);
+        int updatedCount = 0;
+
+        foreach (var packageResult in packagesToUpdateResult)
+        {
+            logger.LogInformation($"    " + Format.PackageUpdate_UpdatedMessage(packageResult.Package.Id, packageResult.Package.CurrentVersion.ToShortString(), packageResult.Package.NewVersion.ToShortString()));
```

Comment:
nit/question: should we be using /t instead?

Reply:
do you mean tabs (`\t`) rather than spaces?  Otherwise I don't know what you're referring to.

I don't know how tabs work on command line apps. If it works then we could, but I'm not aware of any tabs vs spaces best practices for command line app output.